mcs/class/System.Private.CoreLib/System/DateTime.Windows.cs

```diff
@@ -0,0 +1,188 @@
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace System
+{
+	public readonly partial struct DateTime
```

Comment:
Please move this to shared instead (together with Interop changes)

Reply:
I already tried that once. It was shot down for performance reasons (internal calls vs. pinvokes).

Reply:
https://github.com/dotnet/coreclr/pull/21848

Reply:
I can give it one more try and use `!CORECLR` like DateTime.Unix.cs ended-up.